src/stream/src/executor/actor.rs

```diff
@@ -45,6 +51,19 @@ impl ActorContext {
         Arc::new(Self {
             id,
             errors: Default::default(),
+            cur_mem_val: Arc::new(0.into()),
+            last_mem_val: Arc::new(0.into()),
+            total_mem_val: Arc::new(TrAdder::new()),
```

Comment:
Is this still used in non-test case?

Reply:
It should only be used in non-test case, not in unit test. Unittest should never use these fields.

Reply:
`total_mem_val ` should be set to the one in the global memory manager in non-test case right? In ut it should be set to a default value?

Reply:
> `total_mem_val` should be set to the one in the global memory manager in non-test case right? 

Yes, they share the Arc.

> In ut it should be set to a default value?

in ut cuz there is no global memory manager, so we do not need to passed in as arguments, so yes only test code should use `create`, others should use `create_with_counter`.